mandos/example.scen.json

```diff
@@ -0,0 +1,266 @@
+{
+    "name": "example scenario file",
+    "comment": "comments are nice",
+    "checkGas": false,
+    "steps": [
+        {
+            "step": "externalSteps",
+            "path": "other.scen.json"
+        },
+        {
+            "step": "setState",
+            "comment": "not much to comment here, but we can",
+            "accounts": {
+                "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b000000000000000000000000": {
+                    "comment": "we can comment on individual account initializations",
+                    "nonce": "0",
+                    "balance": "0xe8d4a51000",
+                    "storage": {},
+                    "code": ""
+                },
+                "``smart_contract_address________s1": {
+                    "nonce": "0x00",
+                    "balance": "23,000",
+                    "storage": {
+                        "0x19efaebcc296cffac396adb4a60d54c05eff43926a6072498a618e943908efe1": "-5",
+                        "``32_byte_key_____________________": "``string___interpreted___as__bytes",
+                        "``serialized_list_example": [
+                            "``component1",
+                            "``component2"
+                        ],
+                        "``serialized_map_example": {
+                            "``field1": "u32:5",
+                            "``field2": [
+                                "``field2elem1",
+                                "u64:0",
+                                "``field2elem3"
+                            ]
+                        }
+                    },
+                    "code": "file:smart-contract.wasm"
+                }
+            },
+            "newAddresses": [
+                {
+                    "creatorAddress": "``creator_addr____________________",
+                    "creatorNonce": "1234",
+                    "newAddress": "``creator_addr____________________"
+                }
+            ],
```

Comment:
Use the new "address:address_name" format instead of the old "fixed size 32-character string"